### PR TITLE
feat(webhooks): allow proxy settings for webhook notifications

### PR DESCRIPTION
**Pull request by RomanJRW**
relates to OPT-6083

### CODE REVIEW

**Comment by RomanJRW** [comment]
@akeller - done 👍 
Let's hold off on merging before https://github.com/camunda/camunda-optimize/pull/4903 gets approved too if that's ok?

File: versioned_docs/version-1.3/self-managed/optimize-deployment/setup/configuration.md
Comment (grlimacan) [inline versioned_docs/version-1.3/self-managed/optimize-deployment/setup/configuration.md:308]:
```suggestion
|webhookAlerting.webhooks.${webhookName}.proxy.enabled|false|Whether a HTTP proxy should be used for requests to the webhook URL.|
```

**Review by grlimacan — DISMISSED** [review]
Generally ok, but some small things need to be corrected, pls take a look at the comments

File: versioned_docs/version-1.3/self-managed/optimize-deployment/setup/configuration.md
Comment (RomanJRW) [reply versioned_docs/version-1.3/self-managed/optimize-deployment/setup/configuration.md:305]: can you double check this? My IntelliJ renderer also looks broken like you describe, but in any other markdown renderer I think it's fine. Did you see it broken on the actual docs page?

**Comment by RomanJRW** [comment]
@akeller - is this good to merge now?

File: versioned_docs/version-1.3/self-managed/optimize-deployment/setup/configuration.md
Comment (akeller) [reply versioned_docs/version-1.3/self-managed/optimize-deployment/setup/configuration.md:305]: I'll check this before merging.

Comment (akeller) [reply versioned_docs/version-1.3/self-managed/optimize-deployment/setup/configuration.md:305]: This looks good to me after doing an `npm run start` locally. This table appears to have no default values, which is fine.

**Pull request merged**
